docs/src/interface-definition.md

```diff
@@ -158,6 +161,40 @@ AtomsCalculators.@generate_interface function AtomsCalculators.forces!(f::Abstra
 end
 ```
 
+### Implementing Hessian
+
+Hessian can be implemented in similar ways. Using either direct call
+
+```julia
+AtomsCalculators.@generate_interface function AtomsCalculators.hessian(system, calculator::MyType; kwargs...)
+    # we can ignore kwargs... or use them to tune the calculation
+    # or give extra information like pairlist
+
+    # add your own definition
+    l = length(system) * length(position(system, 1))
+    return zeros(l,l) * (u"eV" / u"Å"^2)
```

Comment:
This definition of hessian is inconsistent with how we return forces I believe? Forces are `AbstractVector{<: SVector}` and hence hessians should be `AbstractMatrix{<: SMatrix}`.  The conversion from this format to a `AbstractMatrix{<: Number}` requires the specification of how positions are converted into degrees of freedom.

What is a key difficulty here is that this refers exclusively to the hessian w.r.t. positions. Hessian w.r.t. combined cell and position will need to be discussed.